tests/test_generators/test_pythongen.py

```diff
@@ -68,6 +68,11 @@ def test_multiline_stuff(input_path):
     )
 
 
+def test_ifabsent_stuff(input_path):
```

Comment:
also great! this is the correct test. Ideally it would be named something that is a bit more informative about what it tests, like `test_enum_permissiblevalue_ifabsent` , but whatever, it is in fact testing `ifabsent_stuff`.

Reply:
It tests 2 things: that the resulting Python code (otherwise `make_python` would fail) can be compiled and test_enum_permissiblevalue_ifabsent. That is why I left that name. I will add some comments and rename it.